### PR TITLE
Create first unit test with YAML input

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
This PR creates a unit test that will create a map scatter plot from a IODA netCDF file based on options in an input YAML file.

The PR also contains the functions that the unit test calls.

The next step will be making a stand alone executable script that functions much like this unit test.

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
Screen Shot 2021-11-30 at 1 17 45 PM

The unit test creates a figure like this.

File: src/plotioda/plots/plots.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    mymap.add_title(label=plot.get('title', ''),
+                    loc='left', fontsize=12)
+    mymap.add_title(label=plot.get('cycle string', ''),
+                    loc='right', fontsize=12,
+                    fontweight='semibold')
+    mymap.add_xlabel(xlabel='Longitude')
+    mymap.add_ylabel(ylabel='Latitude')
+
+    # add a lat/lon grid if requested
+    if 'grid' in plot and plot['grid']:
```
Comment (kevindougherty-noaa) [inline src/plotioda/plots/plots.py:66]: So would grid just be added to the yaml as an input if the user wanted it?

Comment (kevindougherty-noaa) [reply src/plotioda/plots/plots.py:66]: Also, I found that this style of grid is way less intrusive on the plot if you would want to test and see:

`mymap.add_grid(linewidth=0.5, color='grey', linestyle='--')`

**Review by kevindougherty-noaa — APPROVED** [review]
Looks really good to me. Easy to follow workflow. Great job.

File: src/plotioda/plots/plots.py (hunk 1 above)
Comment (CoryMartin-NOAA) [reply src/plotioda/plots/plots.py:66]: Yes, the YAML would say `grid: true`, let me add that suggested color for the grid and push a new commit.

**Review by kevindougherty-noaa — APPROVED** [review]

**Pull request merged**